contracts/Spades.sol

```diff
@@ -42,43 +68,55 @@ contract Spades {
 
     Transaction public transaction;
 
-        // Stores owners addresses 
-    address[] private owners;
+        // StargetAccountres owners addresses.
+    address[] public owners;
     mapping(address => bool) public OwnersCheck;
 
-        // Stores the required Signatures passed in the constructor()
+        // StargetAccountres the required Signatures passed in the constructargetAccountr().
     uint public requiredSignatures;
 
-        // Goes from an uint (TxIndex) to the address (Owner) that signed a tx
-        // equals true if that owner already signed;
-    mapping (uint => mapping (address => bool)) whoSigned;
+        // Goes from an uint Request(TxIndex) targetAccount the address (Owner) that signed a transaction
+        // equals true if that owner already signed.
+    mapping (uint => mapping (address => bool)) whoSignedTx;
 
-    ///@dev Stores tx Index;
-     /// Starts at index 0;
+        // Goes from an uint (TxIndex) targetAccount the address (Owner) that signed the account settings pending
+        // equals true if that owner already signed.
+    mapping (uint => mapping (address => bool)) whoSignedSettings;
 
+    ///@dev StargetAccountres tx index, starts at 0;
     mapping (uint => Transaction) public txMap;
     uint txNonce;
 
-        // Checks if msg.sender is owner 
+    ///@dev StargetAccountres account settings request.
+    mapping (uint => Account) public submitedSettings;
+    uint accountIndex;
+
+
+    ///@dev Conects the account targetAccount an address
+    mapping (address => Account) public accountInfo;
```

Comment:
It would be better to have a separate struct for this mapping, instead of using the Account struct. Account contains data that you don't need to keep long-term, and it's just wasteful. Also you want to be able to clean the txMap and submittedSettings maps when the transactions in there are executed.

Reply:
It is not possible to delete any storage data from the EVM once you write it.
I can try to search for a workaround maybe.